File: baekjoon/3036/3036.py
'''
링 python 풀이
https://doctcoder.tistory.com/entry/%EB%A7%81-%ED%92%80%EC%9D%B4
'''
import math

n=int(input())
arr=list(map(int,input().split()))
for i in range(1,n):
	gcds=math.gcd(arr[0],arr[i])
	print(f'{arr[0]//gcds}/{arr[i]//gcds}')

File: baekjoon/3036/3036.cpp
/*
링 C++ 풀이
https://doctcoder.tistory.com/entry/%EB%A7%81-%ED%92%80%EC%9D%B4
*/
#include <iostream>
#include <vector>
#include <numeric>

using namespace std;

int main()
{
	int n;
	cin>>n;
	vector<int>v(n);
	for(int i=0;i<n;i++) cin>>v[i];
	for(int i=1;i<n;i++)
	{
		int g=gcd(v[0],v[i]);
		cout<<v[0]/g<<'/'<<v[i]/g<<"\n";
	}
}
